TestHost/Program.cs

```diff
@@ -135,12 +128,16 @@ private static void DiscoverUnitTests(IEnumerable<ITestDiscoverer> tds, string s
                 timeFilter,
                 (string assemblyPath) =>
                 {
-                    var asmPath = FilePath.NewFilePath(assemblyPath);
                     var disc = new XUnitTestDiscoverer();
                     disc.TestDiscovered.AddHandler(
                         new FSharpHandler<TestCase>(
                             (o, ea) =>
                             {
+                                if (ignoredTests.Any(x => ea.FullyQualifiedName.StartsWith(x)))

```

Comment:
:P XUnitTestDiscoverer is a misnomer. It is good enough to handle all test adapters (xunit, nunit, whatever else comes in the future). I thought there would be different classes for each adapter so I named it Xunit specific.